great_expectations/self_check/util.py

```diff
@@ -164,6 +170,8 @@
         pass
 except ImportError:
     try:
+        # noinspection PyUnresolvedReferences
+        # noinspection PyUnresolvedReferences
```

Comment:
Should this be listed twice?

Reply:
@cdkini Looks like `black` did it (fixing).